### PR TITLE
Implemented argmax in paddle.tensor.search

### LINKED ISSUE #16492: argmax

### PR DESCRIPTION
**Pull request by KevinUli**
Close #16492

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+)
+def test_paddle_argmax(
+    dtype_x_and_axis,
+    keepdim,
+    frontend,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axis = dtype_x_and_axis
+    helpers.test_frontend_function(
```
Comment (xoiga123) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py:27]: Please test for `dtype` as well :pleading_face:

**Review by xoiga123 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py (hunk 1 above)
Comment (KevinUli) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py:27]: I've skipped `dtype` test and added a comment. Thank you 😃

Comment (xoiga123) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py:27]: Sorry one last nitpick, by below the test I meant somewhere right below the function definition line :sweat_smile: Sorry should've worded that better

Comment (KevinUli) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py:29]: Sorry, I misunderstood. Would this be better?

**Review by xoiga123 — APPROVED** [review]

**Pull request merged**